_config.php

```diff
@@ -1,6 +0,0 @@
-<?php
```

Comment:
An empty config.yml is more efficient, and so that's better. Also, _config.php is kind-of-deprecated-unless-you-really-need it.

`type: silverstripe-module` would be nice — but it's a nice-to-have and not a beta1 blocker IMO ;)